app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -33,12 +33,14 @@ def perform(_now)
     )
 
     started_at = Time.zone.now
+    enrollments.update(last_batch_claim_at: started_at)
+    enrollments = InPersonEnrollment.needs_usps_status_check_batch(started_at) if
+      enrollments.size > 0
```

Comment:
I find this line break very surprising, I think the block form would be clearer here
```suggestion
    enrollments = if enrollments.size > 0
      InPersonEnrollment.needs_usps_status_check_batch(started_at) 
    end
```